bridge/test/nftbridge/NFTBridge_test.js

```diff
@@ -316,6 +357,11 @@ contract("Bridge NFT", async function(accounts) {
           });
           utils.checkRcpt(receipt);
 
+          const setFixedFee = await this.NFTBridge.getFixedFee();

```

Comment:
maybe I'd rename to `currentFixedFee` to not confuse this with a setter (somehow)

Reply:
done at https://github.com/rsksmart/tokenbridge/pull/268/commits/9c4e1c7b64b7379ff804bb3f768eed9c96a7461b